src/main/scala/edu/ie3/simona/main/RunSimona.scala

```diff
@@ -68,6 +70,21 @@ trait RunSimona[T <: SimonaSetup] extends LazyLogging {
     val randIdx = rand.nextInt(myWords.length)
     logger.info(myWords(randIdx))
     logger.info("Goodbye!")
+
+    if (!successful) {
+      // to ensure that the link to the log is printed last
+      Thread.sleep(1000)
+
+      val root: Path = Path.of(
+        new File(".").getAbsoluteFile.getParent,
+        "logs",
+        "simona",
+        "simona.log",
+      )
+      logger.error(
+        s"Simulation stopped due to the occurrence of an error! The full log can be found here: $root"
+      )
+    }
```

Comment:
This should take into account the actual output directory that was specified in the configuration. It seems like the path you are printing is relative to simona not based on the configuration